endpoints/openrtb2/video_auction.go

```diff
@@ -478,6 +478,11 @@ func mergeData(videoRequest *openrtb_ext.BidRequestVideo, bidRequest *openrtb.Bi
 			Gender:   videoRequest.User.Gender,
 			Keywords: videoRequest.User.Keywords,
 		}
+		if videoRequest.User.Gdpr.ConsentRequired {
```

Comment:
What does `User.Gdpr.ConsentRequired` mean?

There is a boolean GDPR setting in the IAB spec at `Regs.ext.gdpr` which in which indicates if the request is subject to GDPR regulations. If that's the same meaning here, consider mapping `User.Gdpr.ConsentRequired` to `Regs.ext.gdpr` and `videoRequest.User.Gdpr.ConsentString` to `User.ext.consent`.

Reply:
Updated per our discussion. The video request has been updated to use OpenRTB objects instead of custom ones for User and Video. The User here will instead by copied directly from the request. Handling of `User.Ext.Consent` will be considered downstream.